change autodetect to false in partner_pipeline

cayod opened the pull request:
This PR changes the autodetect in `partner_pipeline_dag` to false to keep the operator running the function `read_local_schema`. The schema with the new `batch_insert_ts` field is not being used.

sydneynotthecity commented:
@cayod will this fix the prod errors we saw yesterday? Running the MGI load job was giving the error:
```
"Error while reading data, error message: CSV table references column position 40, but line contains only 40 columns.; line_number: 2 byte_offset_to_start_of_line: 834 column_index: 40 column_name: "batch_insert_ts" column_type: TIMESTAMP File: gs://ext-partner-sftp/mgi/stellar_transaction_extract_2023120209001487_24707733.csv"
```
Example log [here](https://console.cloud.google.com/logs/query;cursorTimestamp=2023-12-04T18:41:05.003228Z;duration=P2D;query=severity%3E%3DERROR%0A--Hide%20similar%20entries%0A-%2528protoPayload.methodName%3D%22jobservice.jobcompleted%22%2529%0A--End%20of%20hide%20similar%20entries%0A--Hide%20similar%20entries%0A-%2528protoPayload.methodName%3D%22jobservice.getqueryresults%22%2529%0A--End%20of%20hide%20similar%20entries%0ASEARCH%2528%22giving%20up%22%2529%0Atimestamp%3D%222023-12-04T18:41:05.003228Z%22%0AinsertId%3D%22txjec6rku%22?project=hubble-261722)

I removed the `batch_insert_ts` from the schema uploaded to prod Airflow because that field isn't present on the file we receive from MGI, it should just be a defaulted record on the final table.

cayod commented:
@sydneynotthecity you are right this is the best way to deal with this error, I removed the batch_insert_ts in the schema file and added to the ddls scripts the field for adding the column when needed

sydneynotthecity reviewed (approved):
LGTM
